papermage/magelib/box.py

```diff
@@ -73,6 +76,17 @@ def from_xy_coordinates(
     def xy_coordinates(self) -> Tuple[float, float, float, float]:
         return self.l, self.t, self.l + self.w, self.t + self.h
 
+    def __eq__(self, other: Any) -> bool:
+        if not isinstance(other, Box):
+            return False
```

Comment:
if comparing not the same type, can it error out instead?

Reply:
returning false is default behavior in Python! comparisons usually don't raise errors.